pkg/apis/aerogear/v1alpha1/register.go

```diff
@@ -0,0 +1,16 @@
+// +k8s:deepcopy-gen=package,register
+// +groupName=mobile.k8s.io
+package v1alpha1
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/runtime/scheme"
+)
+
+var (
+	// SchemeGroupVersion is group version used to register these objects
+	SchemeGroupVersion = schema.GroupVersion{Group: "mobile.k8s.io", Version: "v1alpha1"}
```

Comment:
The `GroupVersion` should been changed to work with the lasted version of MDC. This should be `mdc.aerogear.org` not `mobile.k8s.io`.